x-pack/plugins/infra/public/alerting/log_threshold/components/expression_editor/log_view_switcher.tsx

```diff
@@ -0,0 +1,29 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import React from 'react';
+import { i18n } from '@kbn/i18n';
+import { EuiFlexItem, EuiFlexGroup, EuiExpression } from '@elastic/eui';
+import { ResolvedLogView } from '../../../../../common/log_views';
+
+const description = i18n.translate('xpack.infra.logs.alertFlyout.logViewDescription', {
+  defaultMessage: 'Log View',
+});
+
+interface LogViewSwitcherProps {
+  logView: ResolvedLogView;
+}
+
+export const LogViewSwitcher: React.FC<LogViewSwitcherProps> = ({ logView }) => {
```

Comment:
Why is this called `LogViewSwitcher` - this is the component planned to choose from different log views right (which is currently not implemented)?

Reply:
I called this way because in future it will allow switching between the available LogViews, while right now is a read-only field.
I mentioned it in the PR description, but I'll make sure to also state it with a code comment and I'll make it clearer in the PR description.